File: Test_Coverage/processed_dataset/tests/test_HumanEval_76.py
from functions.HumanEval_76 import is_simple_power

def test_HumanEval_76():
 
     # Check some simple cases
     assert is_simple_power(16, 2)== True, "This prints if this assert fails 1 (good for debugging!)"
     assert is_simple_power(143214, 16)== False, "This prints if this assert fails 1 (good for debugging!)"
     assert is_simple_power(4, 2)==True, "This prints if this assert fails 1 (good for debugging!)"
     assert is_simple_power(9, 3)==True, "This prints if this assert fails 1 (good for debugging!)"
     assert is_simple_power(16, 4)==True, "This prints if this assert fails 1 (good for debugging!)"
     assert is_simple_power(24, 2)==False, "This prints if this assert fails 1 (good for debugging!)"
     assert is_simple_power(128, 4)==False, "This prints if this assert fails 1 (good for debugging!)"
     assert is_simple_power(12, 6)==False, "This prints if this assert fails 1 (good for debugging!)"
 
     # Check some edge cases that are easy to work out by hand.
     assert is_simple_power(1, 1)==True, "This prints if this assert fails 2 (also good for debugging!)"
     assert is_simple_power(1, 12)==True, "This prints if this assert fails 2 (also good for debugging!)"
 

File: Test_Coverage_Java/src/test/java/test_HumanEval_76.java
import org.junit.Test;
import static org.junit.Assert.*;

public class HumanEval76Test {

    @Test
    public void test_HumanEval_76() {
 
        // Check some simple cases
        assertTrue(HumanEval_76.isSimplePower(16, 2));
        assertFalse(HumanEval_76.isSimplePower(143214, 16));
        assertTrue(HumanEval_76.isSimplePower(4, 2));
        assertTrue(HumanEval_76.isSimplePower(9, 3));
        assertTrue(HumanEval_76.isSimplePower(16, 4));
        assertFalse(HumanEval_76.isSimplePower(24, 2));
        assertFalse(HumanEval_76.isSimplePower(128, 4));
        assertFalse(HumanEval_76.isSimplePower(12, 6));
 
        // Check some edge cases that are easy to work out by hand.
        assertTrue(HumanEval_76.isSimplePower(1, 1));
        assertTrue(HumanEval_76.isSimplePower(1, 12));
    }
}